core/include/detray/masks/annulus2D.hpp

```diff
@@ -168,6 +173,12 @@ class annulus2D {
         return ((r_mod2 >= minR_tol * minR_tol) and
                 (r_mod2 <= maxR_tol * maxR_tol));
     }
+
+    template <typename param_t>
+    DETRAY_HOST_DEVICE inline typename param_t::point2 to_measurement(
```

Comment:
Is this method only needed in the simulation or does it have another usecase?

Reply:
You are right. I removed the function and moved it to `measurement_smearer`